common/src/main/scala/com/gu/multimedia/storagetier/vidispine/VidispineCommunicator.scala

```diff
@@ -1,23 +1,21 @@
 package com.gu.multimedia.storagetier.vidispine
-
+import com.gu.multimedia.storagetier.messages.VidispineField;
 import akka.actor.ActorSystem
 import akka.http.scaladsl.Http
-import akka.http.scaladsl.model.{ContentTypes, HttpEntity, HttpMethods, HttpRequest, MediaRange, MediaRanges, MediaTypes}
+import akka.http.scaladsl.model._
 import akka.http.scaladsl.model.headers.{Accept, Authorization, BasicHttpCredentials}
 import akka.stream.Materializer
-import akka.stream.scaladsl.{Keep, Sink, Source, StreamConverters}
-import akka.util.ByteString
-import org.slf4j.{LoggerFactory, MDC}
-import io.circe.generic.auto._
+import akka.stream.scaladsl.{Keep, StreamConverters}
+import cats.implicits._
 import com.gu.multimedia.storagetier.utils.AkkaHttpHelpers
 import com.gu.multimedia.storagetier.utils.AkkaHttpHelpers.{RedirectRequired, RetryRequired, consumeStream, contentBodyToJson}
+import io.circe.generic.auto._
+import org.slf4j.{LoggerFactory, MDC}
 
-import scala.concurrent.duration._
-import scala.concurrent.duration.FiniteDuration
+import java.net.URLEncoder
+import scala.concurrent.duration.{FiniteDuration, _}
```

Comment:
`import scala.concurrent.duration._` is equivalent

Reply:
That ain't be, bruv. I.e. that is some IntelliJ auto-import.. :) I'll change it back nonetheless.